releasenotes/notes/33203.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+  - 33203
+releaseNotes:
+  - |
+    **Fixed** istio-proxy on virtual machine failed to bootstrap using cient cert after restarting.
```

Comment:
I am not sure what this PR is solving or why this is accurate. OUTPUT_CERTS is already set by default in istio-start.sh. So how is this fixing the issue linked? And why do we need to make it customizable? I would not expect many to want to customize this

Reply:
I may misunderstand the issue. It is not caused that we do not output generated certs for VM but because the special environment setting. I agree that there will not be many need to customize this. So let's just remove the release notes here as it is inaccurate.